certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -2318,10 +2367,117 @@ PROCEDURE EXPIREORDELETEXFERS
            SET EXPIRATIONDATE TO SYSTEMDATE
           END
         END
+       BNODEBUGMSG="SHARE XFER TYPE 2 DELETE/EXPIRE FMERROR='"+FMERROR+"'"
+       CALL BNODEBUGMSGADD
        TMPLOOP=TMPLOOP+1
       END
     END
   END
+ FOR TMPLOOP=1 TO STRANSFERCOUNT
```

Comment:
Is this initialization loop necessary?

Reply:
I kinda like the idea of going into a procedure which will be assigning values with a properly initialized array. That being said, I did move it to the top of the procedure where it really belongs. If you do prefer though, I will remove it.